src/Command/Setup.php

```diff
@@ -81,7 +84,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
     public function validateVar($value)
     {
         if (!preg_match('/^([-A-Z0-9])*$/i', $value)) {
-            throw new RuntimeException(sprintf('This value must contain only alphanumeric characters and hyphens.', $value));
+            throw new \InvalidArgumentException(sprintf(
```

Comment:
Fixed the exception according to the doc block.

But also suggest to use multilined sprintf declaration for exceptions when the exception construct exceeds 120 chars.
Most of the time, it is sufficient to make the php code sniffer happy, and help for readability :)